From 99960783ffa6a63a1efa00b4ec5b300bebc10127 Mon Sep 17 00:00:00 2001
From: gaurav rana <gaurav.rana@freescale.com>
Date: Thu, 26 Mar 2015 15:52:47 +0530
Subject: [PATCH] powerpc/T104xD4: Add Secure boot support for T104xD4RDB
 platforms

defconfig files are added and SFP version for these platforms
is updated.

Signed-off-by: Gaurav Rana <gaurav.rana@freescale.com>
Reviewed-by: York Sun <yorksun@freescale.com>
---
 arch/powerpc/include/asm/fsl_secure_boot.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/powerpc/include/asm/fsl_secure_boot.h b/arch/powerpc/include/asm/fsl_secure_boot.h
index e74698d..f1c95fa 100644
--- a/arch/powerpc/include/asm/fsl_secure_boot.h
+++ b/arch/powerpc/include/asm/fsl_secure_boot.h
@@ -37,7 +37,9 @@
 	defined(CONFIG_T2080QDS) || \
 	defined(CONFIG_T2080RDB) || \
 	defined(CONFIG_T1040QDS) || \
+	defined(CONFIG_T104xD4QDS) || \
 	defined(CONFIG_T104xRDB) || \
+	defined(CONFIG_T104xD4RDB) || \
 	defined(CONFIG_TQMT1042) || \
 	defined(CONFIG_PPC_T1023) || \
 	defined(CONFIG_PPC_T1024)
-- 
1.9.1

